### PR TITLE
New HomeObjects-3K dataset

### PR DESCRIPTION
**Pull request by RizwanMunawar**
@glenn-jocher Hi, this PR adds a new dataset, HomeObjects-3K in [Ultralytics docs](https://docs.ultralytics.com/), Thanks.

### Train

```python
from ultralytics import YOLO

# Load a model
model = YOLO("yolo11n.pt")  # load a pretrained model (recommended for training)

# Train the model
results = model.train(data="HomeObjects-3K.yaml", epochs=100, imgsz=640)
```

#### Inference results

living_room_1p (374)

#### Train batch Image

homeobjects-3k-dataset-sample

## 🛠️ PR Summary

<sub>Made with ❤️ by [Ultralytics Actions](https://github.com/ultralytics/actions)<sub>

### 🌟 Summary
A new indoor object detection dataset, HomeObjects-3K, has been added to Ultralytics, making it easier to train and evaluate models like YOLO11 on common household items. 🏠✨

### 📊 Key Changes
- **New Dataset Added:** Introduced HomeObjects-3K, featuring ~3,000 annotated images of 12 household object classes (e.g., bed, sofa, TV, laptop).
- **Documentation:** Comprehensive docs page created, including usage instructions, sample images, and FAQs.
- **Dataset Listing:** HomeObjects-3K now appears in the supported datasets index and navigation menus.
- **YAML Config:** Added a ready-to-use YAML configuration file for seamless integration with Ultralytics training workflows.

### 🎯 Purpose & Impact
- **Empowers Indoor AI Projects:** Enables users to train, test, and benchmark models for smart home, robotics, AR, and educational applications.
- **Beginner-Friendly:** Clean labels and YOLO-compatible format make it accessible for students and hobbyists.
- **Accelerates Development:** Provides a high-quality, open-source dataset for rapid prototyping and research in indoor object detection.
- **Easy Integration:** Simple training commands and clear documentation help users get started quickly with YOLO11 and other Ultralytics models.

🚀 This update broadens the scope of Ultralytics datasets, making it even easier to build intelligent indoor vision solutions!

### CODE REVIEW

**Comment by UltralyticsAssistant** [comment]
👋 Hello @RizwanMunawar, thank you for submitting this 🚀 dataset addition PR to `ultralytics/ultralytics`! This is an automated response to help streamline your contribution. An Ultralytics engineer will review and assist you soon.

Please review the checklist below to ensure a smooth integration:

- ✅ **Define a Purpose**: Clearly explain the goal of your dataset addition and reference any [related issues](https://github.com/ultralytics/ultralytics/issues) in your PR description. Be sure your commit messages are clear and follow project standards.
- ✅ **Synchronize with Source**: Make sure your PR is up to date with the `ultralytics/ultralytics` `main` branch. If needed, update by clicking 'Update branch' or by running `git pull` and `git merge main` locally.
- ✅ **Ensure CI Checks Pass**: Confirm all Ultralytics [Continuous Integration (CI)](https://docs.ultralytics.com/help/CI/) checks are passing. Please resolve any issues if CI fails.
- ✅ **Update Documentation**: Your PR updates [documentation](https://docs.ultralytics.com/) for the new dataset—thank you! Please double-check for completeness and consistency.
- ✅ **Add Tests**: If relevant, include or update tests for your changes, and verify that all tests pass.
- ✅ **Sign the CLA**: If this is your first Ultralytics PR, please sign our [Contributor License Agreement (CLA)](https://docs.ultralytics.com/help/CLA/) by commenting: "I have read the CLA Document and I sign the CLA".
- ✅ **Minimize Changes**: Limit your changes to the **minimum** necessary for this dataset integration. _"It is not daily increase but daily decrease, hack away the unessential. The closer to the source, the less wastage there is."_ — Bruce Lee

For detailed contribution guidelines, please see our [Contributing Guide](https://docs.ultralytics.com/help/contributing/).

If you have questions or need clarification, feel free to comment below. Thank you for helping expand the Ultralytics ecosystem—your HomeObjects-3K dataset will enable exciting new indoor vision applications with YOLO11 and beyond! 🏠📦✨

**Comment by glenn-jocher** [comment]
@RizwanMunawar this is amazing, great work! PR looks good :)

**Review by glenn-jocher — COMMENTED** [review]
Approved!

**Comment by UltralyticsAssistant** [comment]
🎉 Fantastic work, @RizwanMunawar and @glenn-jocher! The addition of HomeObjects-3K is a major leap forward for Ultralytics indoor vision projects—empowering both beginners and experts to innovate with ease. As Helen Keller said, “Alone we can do so little; together we can do so much.” Your collaborative effort and attention to detail are opening new doors for the community to explore, learn, and create with YOLO11 and Ultralytics HUB. Thank you for your outstanding contribution! 🚀

**Comment by glenn-jocher** [comment]
The HomeObjects-3K dataset is a valuable addition to our ecosystem. For those interested in trying it out, the dataset includes 2,285 training images and 404 validation images across 12 common household object classes. It's particularly useful for indoor scene understanding applications like smart home automation, robotics navigation, and AR. You can start training immediately with `data="HomeObjects-3K.yaml"` which will automatically download the dataset (~390MB). The YAML configuration is already included in the latest Ultralytics package, making integration seamless.
